test_cases/stdlib/builtins/test_tuple.py

```diff
@@ -1,5 +1,5 @@
-from typing_extensions import assert_type
 from typing import Tuple
+from typing_extensions import assert_type
```

Comment:
Weird, I wonder how this passed on my PR.

Reply:
It surprised me too :)